### PR TITLE
Support for Alchemy and Ankr + Refactor

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: src/utils/sentry/index.ts [hunk 1]
```diff
@@ -15,5 +15,5 @@ export const sentryTracker = new BrowserClient({
   // We recommend adjusting this value in production
   tracesSampleRate: 1.0,
   // do not edit this, it is subject to be changed automatically by scripts/release.js
-  release: '<VERSION>',
+  release: 'v1-sdk@1.10.0-d28fd6e',
```
Comment (ssbarbee) [inline src/utils/sentry/index.ts:18]: revert this. You are probably running builds locally

File: src/utils/errors/errorMapping.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "price outside of range": "Internal error: Price outside of range in TickMath",
+    "Wad Ray Math: 49": "Internal error: addition overflow in WadRayMath",
+    "Wad Ray Math: 50": "Internal error: division by zero in WadRayMath",
+    "PositionNetZero": "No need to unwind a net zero position",
+    "CannotLiquidate": "Position is not liquidatable",
+    "OnlyOwnerCanUpdatePosition": "No approval to update this position",
+    "OnlyVAMM": "No approval for this operation",
+    "OnlyFCM": "No approval for this operation",
+    "CannotSettleBeforeMaturity": "Cannot settle before maturity",
+    "NotEnoughFunds": "No enough funds to perform this operation",
```
Comment (ssbarbee) [inline src/utils/errors/errorMapping.json:70]:
```suggestion
    "NotEnoughFunds": "Not enough funds to perform this operation",
```

can you double check all?

Comment (0xZenus) [reply src/utils/errors/errorMapping.json]: https://github.com/Voltz-Protocol/v1-sdk/pull/132/commits/a2e1124cf0eb84af5b18f0253497806f52030238

File: src/utils/sentry/index.ts (hunk 1 above)
Comment (0xZenus) [reply src/utils/sentry/index.ts]: https://github.com/Voltz-Protocol/v1-sdk/pull/132/commits/e7f9f39f084d544772329f7821ec4878cfd5c4fc

**Review by ssbarbee — APPROVED** [review]

**Pull request merged**